### ISSUE TITLE
[IT Support] Change user Enrollment Organization Group - CTM Ticket - INC0219138

### ISSUE THREAD

**Issue by dianamartin** [cityofaustin/atd-data-tech#10010]
Submitted a ServiceDesk ticket `INC0219138`

Wrote on 8/11
>I have a few users in my AirWatch console that are listed in the wrong Enrollment Organization Group, they are listed as being in the "Communication and Technology Management ", they need to be in the "ATD - Parking Meters" group. I'm trying to deploy new ipad mini to this operational group and my normal contact, Danny C. is out of office. I just don't have the admin credentials to change that because it's outside of the ATD groups, which I'm an admin for.

For users: 
- Hector A
- Melton V
- Jacob C

**Comment by dianamartin** [comment]
Emailed stakeholders to let them know of the status 8/11
>I looked at the user list and 
-	Hector A
-	Melton V
are in the wrong group, in a CTM Enrollment Group and I have to put in a CTM help desk ticket to get that corrected. 

With “Steven S”, what is his username? I couldn’t find him in the system. 





![image](https://user-images.githubusercontent.com/21112975/184255681-30442ee8-874f-4f6b-a8e3-baa0954503dc.png)

**Comment by dianamartin** [comment]
Received email 8/15 from Michael S. (CTM) that he had added Hector and Jacob to the correct group. I checked it and Melton was the only one in the wrong group. I asked that the ticket be reopened and corrected. 

I also asked that Steven S. be added to the correct group bc he was the only other person that wasn't in the list that should have been. Michael corrected that last follow-up request I had. I confirmed the ticket to be resolved. 

Emailed stakeholders 8/15
>The CTM ticket is closed, they were able to add Hector, Melton and you to the correct group and also correct Steven to be added to the Parking Meters group too. 

>Let me know if you need anything else!
